### PR TITLE
Integration Part 6 (FINAL)

### PR DESCRIPTION
**Pull request by afnanee213**
-Final integration with database
-Change password system reworked
-Password reset updated
-Menu system updated
-Cancel order from Student side

### CODE REVIEW

**Comment by afnanee213** [comment]
-Updated the look of the tables for each page.

**Comment by FrostedOrchard** [comment]
This should be approved

File: WarriorDeliveryWebsite/DiningDelivery2/ChangePasswordDelivererSQL.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    
+    echo "<h2> Incorrect Credentials</h2>";
+    
+}
+
+?>
+
+
+
+
```
Comment (neelesh-wayne) [inline WarriorDeliveryWebsite/DiningDelivery2/ChangePasswordDelivererSQL.php:57]: Excess spacing, should remove

File: WarriorDeliveryWebsite/DiningDelivery2/CreateAccount.php [hunk 2]
```diff
@@ -53,15 +53,32 @@
...
+   ';
    
+   $headers = 'From:wsudiningdelivery@gmail.com' . "\r\n";
+   mail($myemail, $subject, $message, $headers);
+	 
+	 
+	  }
+   
+	 
+
```
Comment (neelesh-wayne) [inline WarriorDeliveryWebsite/DiningDelivery2/CreateAccount.php:79]: Too much extra space, remove

**Comment by SanazRabinia1** [comment]
In this version we removed redundant files.

File: WarriorDeliveryWebsite/DiningDelivery2/ForgotPasswordDeliverer.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  
+
+
+
+<br>
+
+
+<h2> Your password has been reset. An email has been set the to the registered email containing the new password.</h2>
+
+
```
Comment (neelesh-wayne) [inline WarriorDeliveryWebsite/DiningDelivery2/ForgotPasswordDeliverer.php:56]: Remove extra padding

**Pull request merged**